tutorials/math/random_number_generation.rst

```diff
@@ -315,6 +320,29 @@ We can apply similar logic from arrays to dictionaries as well:
         # The same metal may be selected multiple times in succession.
         return random_metal
 
+ .. code-tab:: csharp
+
+    private var metals = new Godot.Collections.Dictionary
+    {
+        {"copper", new Godot.Collections.Dictionary{{"quantity", 50}, {"price", 50}}},
+        {"silver", new Godot.Collections.Dictionary{{"quantity", 20}, {"price", 150}}},
+        {"gold", new Godot.Collections.Dictionary{{"quantity", 3}, {"price", 500}}}
+    };
+
+    public override void _Ready()
+    {
+        GD.Randomize();
+        for (int i = 0; i < 20; i++)
+        {
+            GD.Print(GetMetal());
+        }
+    }
+
+    public Godot.Collections.Dictionary GetMetal()
+    {
+        var randomMetal = metals.Values()[GD.Randi() % metals.Length];
```

Comment:
This method doesn't return.
This line wouldn't compile for several reasons:
- `.Values` is a property, not a method
- I don't think the return of `.Values` can be `[]`-indexed
- Godot collections use `.Count` and not `.Length`